src/edu/csus/ecs/pc2/shadow/IRemoteContestAPIAdapter.java

```diff
@@ -40,10 +40,14 @@ public interface IRemoteContestAPIAdapter {
      * Returns an {@link InputStream} from which CLICS events from a remote
      * contest can be read.
      * 
+     * @param token the token signifies a starting  point in the event feed.
+     *      this is done by adding a url parameter ?since_token=token
+     *      Passing an empty string or null will start the feed at the beginning
+     * 
      * @return the remote system event feed input stream, or null if no input stream could be acquired
      * 
      */
-    public InputStream getRemoteEventFeedInputStream() ;
```

Comment:
Agreed.  I thought about it, and since only 2 modules used the interface, I decided to change the signature.  In hindsight, it does make sense to have a default constructor.

Reply:
fyi there is one in PR that uses the interface to compare the primary CCS model with the pc2 model.